python-package/lightgbm/basic.py

```diff
@@ -408,6 +408,11 @@ def _label_from_pandas(label):
         if not all(dtype.name in PANDAS_DTYPE_MAPPER for dtype in label_dtypes):
             raise ValueError('DataFrame.dtypes for label must be int, float or bool')
         label = label.values.astype('float')
+    elif isinstance(label, Series):
+        label_dtypes = label.dtypes
+        if not all(dtype.name in PANDAS_DTYPE_MAPPER for dtype in label_dtypes):
```

Comment:
I think pandas.Series.dtypes is not iterable (only one value). Did you test on it? This part is not needed, in simple_example.py I already used Series as label and it works fine.